mama/c_cpp/src/gunittest/c/publishertest.cpp

```diff
@@ -105,15 +159,18 @@ TEST_F (MamaPublisherTestC, CreateDestroy)
  *
  *  Expected Result: MAMA_STATUS_OK
  */
-TEST_F (MamaPublisherTestC, DISABLED_Send)
+TEST_F (MamaPublisherTestC, Send)
```

Comment:
Can you add 'Event' prefix to all unit tests which depend on publisher events to be implement in the bridge (E.g. 'EventSend' here) so that it can be easily filtered out for bridges which don't support this functionality?